### PR TITLE
oncoprint - use max for MUTATION_COUNT attribute instead of average

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

File: src/shared/components/oncoprint/DataUtils.ts [hunk 1]
```diff
@@ -352,21 +352,27 @@ export function fillClinicalTrackDatum(
...
+            if (values.length === 0) {
                 fillNoDataValue(trackDatum, attribute);
             } else {
-                // average
-                trackDatum.attr_val = numValSum / numValCount;
-                trackDatum.attr_val_counts[trackDatum.attr_val] = 1;
+                switch (attribute.clinicalAttributeId) {
+                    case "MUTATION_COUNT":
+                        // max
+                        trackDatum.attr_val = values.reduce((max, nextVal)=>Math.max(max, nextVal), Number.NEGATIVE_INFINITY);
```
Comment (alisman) [inline src/shared/components/oncoprint/DataUtils.ts]: is this different from just Math.max(values) ?

File: src/shared/components/oncoprint/DataUtils.ts [hunk 2]
```diff
@@ -352,21 +352,27 @@ export function fillClinicalTrackDatum(
...
+                        break;
+                    default:
+                        // average
+                        trackDatum.attr_val = values.reduce((sum, nextVal)=>sum+nextVal, 0) / values.length;
```
Comment (alisman) [inline src/shared/components/oncoprint/DataUtils.ts]: _.mean(values) ?

File: src/shared/components/oncoprint/DataUtils.ts (hunk 1 above)
Comment (adamabeshouse) [reply src/shared/components/oncoprint/DataUtils.ts]: was not aware of this

File: src/shared/components/oncoprint/DataUtils.ts (hunk 2 above)
Comment (adamabeshouse) [reply src/shared/components/oncoprint/DataUtils.ts]: didn't know about that

**Pull request merged**